test/js/specs/happy.js

```diff
@@ -1,16 +1,48 @@
 const path = require('path')
+const expect = require('chai').expect;
 import {describe, it} from '../utils/mochaw'
 
 const options = {
   pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload']
 }
 
-describe('Happy Paths',options, ({driver,pageObjects}) => {
+const localhostUrl = 'https://localhost:8080/'
+
+describe('Happy Paths',options, ({driver,$,pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
 
+  //Welcome Screen Tests
+  it('test website title', async () => {
+    await driver.get(localhostUrl)
+    const title = await driver.getTitle();
+    expect(title).to.equal('Onfido SDK Demo');
+  })
+
+  it('test welcome screen title', async () => {
+    const welcomeTitleText = await welcome.welcomeTitle.getText()
+    expect(welcomeTitleText).to.equal('Open your new bank account');
+    const welcomeTitle = await welcome.welcomeTitle.isDisplayed()
```

Comment:
This does not need to be assigned, unless you are planning to call expect after

Reply:
Or am I missing called and these are being exported and called somewhere? 🤔

Reply:
💯 
Done.

Reply:
> Or am I missing called and these are being exported and called somewhere? 🤔

nope, they are used only here